src/components/FeatureAttributes/FeaturesAttributesDependencies/FeaturesAttributesDependencies.stories.tsx

```diff
@@ -149,3 +149,23 @@ export const NoFeatures: Story = {
     paramsAtom: getInferFeatureAttributesParamsAtom({}),
   },
 };
+
+// TODO Goals
```

Comment:
Is there a ticket for this TODO?

Reply:
Not at this time. Related to the general performance concerns in BYOD implementation.